src/activities/space-encounter/SpaceEncounter.qml

```diff
@@ -0,0 +1,178 @@
+/* GCompris - space-encounter.qml
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
```

Comment:
Please update copyrights with your name and email
if it is a new activity (not a port) you can remove this line